app/mailers/candidate_mailer.rb

```diff
@@ -43,57 +43,45 @@ def new_referee_request(reference, reason:)
     )
   end
 
-  def application_rejected_all_rejected(application_choice)
+  def all_applications_rejected(application_choice)
     @course = application_choice.course_option.course
-    @application_choice = application_choice
+    @application_choice = RejectedApplicationChoice.new(application_choice)
     @candidate_magic_link = candidate_magic_link(@application_choice.application_form.candidate)
+    @multiple_applications = application_choice.self_and_siblings.count > 1
 
-    email_for_candidate(
-      application_choice.application_form,
-      subject: I18n.t!(
-        application_choice.rejected_by_default ? 'candidate_mailer.application_rejected_by_default.subject' : 'candidate_mailer.application_rejected.all_rejected.subject',
-      ),
-      template_name: :application_rejected_all_rejected,
-    )
+    email_for_candidate(application_choice.application_form)
   end
 
-  def application_rejected_awaiting_decisions(application_choice)
-    @decisions = application_choice.self_and_siblings.select(&:awaiting_provider_decision?)
-    @application_choice = application_choice
-
-    # We cannot use `through:` associations with FactoryBot's `build_stubbed`. Using
-    # the association directly instead allows us to use `build_stubbed` in tests
-    # and mailer previews.
+  def application_rejected__one_offer_one_awaiting_decision(application_choice)
```

Comment:
Not for this PR but this double underscore notation indicates that `CandidateMailer` is spread over a number of different concerns as we're namespacing via method names. At ~400 lines it's getting to the upper end of easy to work with, perhaps this is something to consider as tech debt as we could no doubt break this down into smaller themes.

Reply:
I agree. There are some quick fixes that can be done as well, like removing assignments to `@application_form` as `email_candidate` already does that. But I'll do a seperate PR as there is already another open PR that touches the candidate mailer.